python312Packages.yoto-api: 1.24.4 -> 1.24.5

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for python312Packages.yoto-api is: A python package that makes it a bit easier to work with the yoto play API.

meta.homepage for python312Packages.yoto-api is: https://github.com/cdnninja/yoto_api

meta.changelog for python312Packages.yoto-api is: https://github.com/cdnninja/yoto_api/releases/tag/v1.24.5


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed
- found 1.24.5 with grep in /nix/store/29zxk8xbvf60cs65g61i3j5rmah6ms8i-python3.12-yoto-api-1.24.5
- found 1.24.5 in filename of file in /nix/store/29zxk8xbvf60cs65g61i3j5rmah6ms8i-python3.12-yoto-api-1.24.5

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
4 total rebuild path(s)

4 package rebuild(s)

First fifty rebuilds by attrpath

home-assistant-custom-components.yoto_ha
python312Packages.yoto-api
python313Packages.yoto-api
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/29zxk8xbvf60cs65g61i3j5rmah6ms8i-python3.12-yoto-api-1.24.5 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A python312Packages.yoto-api https://github.com/r-ryantm/nixpkgs/archive/4bf8dba0e7757aefb94924bae17b5fb795bbf2b2.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/4bf8dba0e7757aefb94924bae17b5fb795bbf2b2#python312Packages.yoto-api
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/29zxk8xbvf60cs65g61i3j5rmah6ms8i-python3.12-yoto-api-1.24.5
ls -la /nix/store/29zxk8xbvf60cs65g61i3j5rmah6ms8i-python3.12-yoto-api-1.24.5/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>home-assistant-custom-components.yoto_ha</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.yoto-api</li>
    <li>python312Packages.yoto-api.dist</li>
    <li>python313Packages.yoto-api</li>
    <li>python313Packages.yoto-api.dist</li>
  </ul>
</details>

---

###### Maintainer pings

cc @seberm for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
python312Packages.yoto-api: 1.24.4 -> 1.24.5

seberm commented:
Just FYI: The yoto-api version must be in sync with `yoto_ha` custom component (https://github.com/NixOS/nixpkgs/pull/373807#issuecomment-2592968928):
- https://github.com/cdnninja/yoto_ha/blob/master/custom_components/yoto/manifest.json

dotlambda committed:
home-assistant-custom-components.yoto_ha: 1.22.4 -> 1.22.6

Diff: https://github.com/cdnninja/yoto_ha/compare/refs/tags/v1.22.4...v1.22.6

Changelog: https://github.com/cdnninja/yoto_ha/releases/tag/v1.22.5
           https://github.com/cdnninja/yoto_ha/releases/tag/v1.22.6
